src/app/content/search/components/SearchResultsSidebar/__snapshots__/index.spec.tsx.snap

```diff
@@ -872,7 +872,7 @@ exports[`SearchResultsSidebar matches snapshot with related key terms 1`] = `
         aria-current={true}
         className="c15 c16"
         data-testid="related-key-term-result"
-        href="books/book-slug-1/pages/test-page-1?query=cool%20search&target=%7B%22index%22%3A0%2C%22type%22%3A%22search%22%7D#test-pair-page1"
```

Comment:
I'm not clear on why the query is removed

Reply:
It isn't set in the test, it was leaking from another one and now it isn't